pyemu/utils/pst_from.py

```diff
@@ -1830,6 +1839,10 @@ def add_parameters(
         comment_char=None,
         par_style="multiplier",
         initial_value=None,
+        prep_pp_hyperpars=False,
+        pp_options={},
```

Comment:
Note to self:
Would be keen to support expression of `pp_space` and `use_pp_zones` in `pp_options` dict. Just need to make sure backcompat and decide on which takes priority if both are present.

Reply:
Good call.  Maybe we can mark pp_space and use_pp_zones as deprecated and just map their values in pp_options for now?

Reply:
yeah, I think that should work. We should make that we document all the possible pp keywords but suspect this will be a moving target for a bit.